### PR TITLE
Added min. planet level to capture

### PR DESCRIPTION
**Pull request by ericet**
To give users more options to capture the planets

### CODE REVIEW

**Review by Stx69 — COMMENTED** [review]
working.

**Review by Stx69 — APPROVED** [review]

**Comment by jacobrosenthal** [comment]
@snowtigersoft care to weigh in on anything, this was your plugin originally

File: content/artifacts/artifactory/plugin.js [hunk 1]
```diff
@@ -245,7 +245,7 @@ function Unfound({ selected }) {
   let [lastLocationId, setLastLocationId] = useState(null);
 
   let planets = myPlanetsWithFindable()
-    .filter(planet => !planet.hasTriedFindingArtifact && !planet.unconfirmedProspectPlanet && planet.prospectedBlockNumber === undefined ) ;
+    .filter(planet => !planet.hasTriedFindingArtifact && (planet.prospectedBlockNumber === undefined || planet.hasOwnProperty('unconfirmedFindArtifact')));
```
Comment (snowtigersoft) [inline content/artifacts/artifactory/plugin.js:248]: Are you sure the planet has property "unconfirmedFindArtifact" when it finish prospect(means prospectedBlockNumber is not undefined) BUT not start find?
If you want to hide the prospect expired planets, I suggest change this line to:
```js
.filter(planet => !planet.hasTriedFindingArtifact && (planet.prospectedBlockNumber === undefined || !prospectExpired(currentBlockNumber, planet.prospectedBlockNumber)))
```

**Review by snowtigersoft — CHANGES_REQUESTED** [review]
If you want to hide the prospect expired planets, I suggest change this line to:
```js
.filter(planet => !planet.hasTriedFindingArtifact && (planet.prospectedBlockNumber === undefined || !prospectExpired(currentBlockNumber, planet.prospectedBlockNumber)))
```

**Review by Stx69 — APPROVED** [review]

**Review by Stx69 — APPROVED** [review]

File: content/artifacts/artifactory/plugin.js (hunk 1 above)
Comment (ericet) [reply content/artifacts/artifactory/plugin.js]: Yes, i'm sure the planet has the property "unconfirmedFindArtifact" after the prospect. 
I think your way is better to hide the prospect expired planets. I've made the change you suggested. Thanks.

**Pull request merged**